sdk/mobile-center/src/main/java/com/microsoft/azure/mobile/ingestion/http/IngestionHttp.java

```diff
@@ -82,6 +82,12 @@
     private static final int READ_TIMEOUT = 20000;
 
     /**
+     * Regex pattern for application secret in header.
```

Comment:
If it's too short, obfuscate everything, I believe that's how we did it for azme gcm on portal.